.github/workflows/build.yml

```
@@ -72,3 +71,31 @@ jobs:
       run: |
         docker push ${{env.DOCKER_REPO}}:${{env.BRANCH_TAG}}
         docker push ${{env.DOCKER_REPO}}:${{env.SHA_TAG}}
+
+    - name: Pin Terraform version
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.12.29
+
+    - name: Install Terraform CloudFoundry Provider
+      run: |
+          mkdir -p $HOME/.terraform.d/plugins/linux_amd64
+          wget -O ${{ env.CF_PROVIDER }} https://github.com/cloudfoundry-community/terraform-provider-cf/releases/download/v0.12.2/terraform-provider-cloudfoundry_linux_amd64
+          chmod +x ${{ env.CF_PROVIDER }}
+      env:
+        CF_PROVIDER: $HOME/.terraform.d/plugins/linux_amd64/terraform-provider-cloudfoundry
+
+    - name: Terraform init
+      run: |
+          echo ${{ env.PAAS_TF_DIR }}
+          cd ${{ env.PAAS_TF_DIR }}
+          terraform init \
+           -backend-config=storage_account_name=${{ env.BACKEND_STORAGE_ACCOUNT_NAME }} \
+           -backend-config=access_key=${{ env.BACKEND_ACCESS_KEY }} \
+           -backend-config=key=${{ env.BACKEND_KEY }}
```

could use this
```suggestion
          terraform init \
           -backend-config=workspace-variables/qa_backend.tfvars
```
`qa` can be substituted with a `${{ env.app_environment }}` if required